tests/MyGame/Example/Monster.lua

```diff
@@ -62,6 +62,14 @@ function Monster_mt:Inventory(j)
     end
     return 0
 end
+function Monster_mt:InventoryAsString(start, stop)
+    local o = self.view:Offset(14)
```

Comment:
we should probably just make a local function (or put it somewhere with the other Lua functions) that does all this logic, since the only difference is the type offset.

```
local function byteString(view, offset, start, stop)
    local o = view:Offset(offset)
    if o ~= 0 then
        local a = view:Vector(o) + start - 1
        return view.bytes:Slice(a, a + stop - start + 1)
    end
    return nil
end
```

Reply:
Updated.